### PR TITLE
1.11

### PR DESCRIPTION
**Pull request by rennerocha**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Release notes
 =============
 
+1.11.0 (2019-08-02)
+-------------------
+
+- Allowed per-field checking in ValidationMonitorMixin
+- Added option to set AWS Region Name on SES E-Mail action
+- Added default Value for 'SPIDERMON_BODY_HTML_TEMPLATE' setting
```
Comment (Gallaecio) [inline CHANGES.rst]:
```suggestion
- Added default value for 'SPIDERMON_BODY_HTML_TEMPLATE' setting
```

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+- Fixed bug in logging of Slack messages when fake setting is enabled
+- Enforce lxml 4.3.5 or lower for Python 3.4
```
Comment (Gallaecio) [inline CHANGES.rst]:
```suggestion
- Enforced lxml 4.3.5 or lower for Python 3.4
```

(to keep the same tense as other statements)

File: setup.py [hunk 3]
```diff
@@ -25,7 +25,7 @@
     package_data={"spidermon": ["VERSION"]},
     zip_safe=False,
     include_package_data=True,
-    install_requires=["jsonschema[format]", "python-slugify", "six>=1.10.2"],
+    install_requires=["jsonschema[format]", "python-slugify", "six>=1.11.0"],
```
Comment (Gallaecio) [inline setup.py]: Bad luck with `bumpversion`?

Comment (rennerocha) [reply setup.py]: Yes. But the current version is 1.12.0, so it is not a big problem.

**Pull request merged**